openquake/server/settings.py

```diff
@@ -1,8 +1,12 @@
 import os
 
+from django.conf.global_settings import TEMPLATE_CONTEXT_PROCESSORS
 from openquake.engine import settings as oqe_settings
 
-DEBUG = False  # NB: when True, test_haz_risk_ok breaks!
+
+OQSERVER_ROOT = os.path.dirname(__file__)
+
+DEBUG = True  # NB: when True, test_haz_risk_ok breaks!
```

Comment:
With `DEBUG = True` a test breaks (see https://ci.openquake.org/job/zdevel_oq-engine/1149/testReport/junit/openquake.server.tests.functional_test/EngineServerTestCase/test_ok/). However with `DEBUG = False` django is unable to serve static content. Before merging we should consider fixing the tests or skip it.